control.js

```diff
@@ -0,0 +1,115 @@
+/* NOTES
+Bit 7 - Not used
+Bit 6 - Not used
+Bit 5 - P15 Select Action buttons    (0=Select)  ROM polls by pulling low    active LOW
+Bit 4 - P14 Select Direction buttons (0=Select)  ROM polls by pulling low    active LOW
+Bit 3 - P13 Input: Down  or Start    (0=Pressed) (Read Only)
+Bit 2 - P12 Input: Up    or Select   (0=Pressed) (Read Only)
+Bit 1 - P11 Input: Left  or B        (0=Pressed) (Read Only)
+Bit 0 - P10 Input: Right or A        (0=Pressed) (Read Only)
+ *
+ *
+ *  
+ *  add this to dan's memory map under I/O registers 
+ *  there is a note about 0BJ registers on page 272
+ *  9 = IME gameboy manual does not state an address, top of page 269
+ *  need to decide whether I need if or else if statement for P14 & P15. probably else if since ROM polls each pin sequentially 
+ */
+
+
+var ControlRegister = []; // control register array
+
+var P14 = 1; // pulling P14 low polls directional buttons
+var P15 = 1; // pulling P15 low polls action buttons
+
+//#defines used for ControlRegister
+const P1    = 0; //address 0xFF00
+const SB    = 1; //address 0xFF01
+const SC    = 2; //address0xFF02
+const DIV   = 3; //address0xFF04
+const TIMA  = 4; //address 0xFF05
```

Comment:
correct numbers to reflect proper hex address, each one is 1 byte. change decimal values to hex